resources/local/chainspec.toml

```diff
@@ -28,7 +28,7 @@ validator_slots = 5
 auction_delay = 3
 # initial era
 initial_era_id = 0
-# Locked funds period
+# Default lock period for new bid entries represented in eras.
```

Comment:
It's not a default, is it?
Maybe something like:
```
The delay for the payout of funds, in eras. If a withdraw request is included in a block in era N
(other than the last one), they are paid out in the last block of era N + locked_funds_period.
```
_If_ that's correct! Maybe I'm missing a `+ 1` here or something, or I'm misinterpreting this setting entirely. (@mpapierski: Please double-check!)

Reply:
@afck you're correct. Your funds would be sent back if N+locked_funds_period

Reply:
Updated comments accordingly